README.md

```diff
@@ -11,10 +11,12 @@ npm i react-turnstile
 ## Usage
 
 ```jsx
-import Turnstile from "react-turnstile";
+import Turnstile, { useTurnstile } from "react-turnstile";
 
 // ...
 
+const { reset } = useTurnstile(); // Access turnstile reset method
```

Comment:
Please actually use the `reset()` function in the example, currently it's just a dead import.
Example: `onLoad={((widgetId) => setTimeout(reset, 30000, widgetId)) /* reset widget after 30s */}`

Reply:
Will do